src/platform/forms/components/PrivacyAgreement.jsx

```diff
@@ -1,28 +1,14 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import Checkbox from '@department-of-veterans-affairs/component-library/Checkbox';
+import { VaPrivacyAgreement } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
```

Comment:
These updates should already be in `main`.  Can you please merge that branch in?